.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+AllCops:
+  TargetRubyVersion: 2.0.0
+
+Documentation:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'features/*'
+    - 'spec/**/*'
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/LineLength:
+  Enabled: false
+
+Metrics/MethodLength:
+  Max: 50
```

Comment:
This feels a bit arbitrary here - I think I'd actually lean towards just disabling.

Reply:
For one off cases, you can add inline comment to disable cops

Reply:
I went with disabling altogether as there quite a few...! Thanks!